sdk/spring/azure-spring-boot-starter-keyvault-certificates/README.md

```diff
@@ -89,7 +184,8 @@ If you are using `RestTemplate` use code similar to the example below.
             .loadTrustMaterial(ks, new TrustSelfSignedStrategy())
             .build();
 
-        SSLConnectionSocketFactory csf = new SSLConnectionSocketFactory(sslContext);
+        HostnameVerifier allowAll = (String hostName, SSLSession session) -> true;
+        SSLConnectionSocketFactory csf = new SSLConnectionSocketFactory(sslContext, allowAll);
```

Comment:
Please add Java sample code in `src/samples/java`.
Refs: https://github.com/Azure/azure-sdk-for-java/blame/master/sdk/spring/azure-spring-boot-starter-active-directory/README.md#L312